src/config/source_map.yaml

```diff
@@ -1 +1,83 @@
 git:
+- name: openvino_master
+  automerge: 'yes'
+  mode: 'branch-to-branch'
+  src:
+    url: https://github.com/openvinotoolkit/openvino.git
+    branch: master
+  dest:
+    url: https://github.com/opendatahub-io/openvino.git
+    branch: master
+- name: openvino_contrib_master
+  automerge: 'yes'
+  mode: 'branch-to-branch'
+  src:
+    url: https://github.com/openvinotoolkit/openvino_contrib.git
+    branch: master
+  dest:
+    url: https://github.com/opendatahub-io/openvino_contrib.git
+    branch: master
+- name: openvino_model_server_main
+  automerge: 'yes'
+  mode: 'branch-to-branch'
+  src:
+    url: https://github.com/openvinotoolkit/model_server.git 
+    branch: main 
+  dest:
+    url: https://github.com/opendatahub-io/openvino_model_server.git 
+    branch: main
+- name: openvino_release_2023.3
+  automerge: 'yes'
+  mode: 'branch-to-branch'
+  src:
+    url: https://github.com/openvinotoolkit/openvino.git
+    branch: releases/2023/3
+  dest:
+    url: https://github.com/opendatahub-io/openvino.git
+    branch: 2023.3-release
+- name: openvino_contrib_release_2023.3
+  automerge: 'yes'
+  mode: 'branch-to-branch'
+  src:
+    url: https://github.com/openvinotoolkit/openvino_contrib.git
+    branch: releases/2023/3 
+  dest:
+    url: https://github.com/opendatahub-io/openvino_contrib.git
+    branch: 2023.3-release
+- name: openvino_model_server_release_2023.3
+  automerge: 'yes'
+  mode: 'branch-to-branch'
+  src:
+    url: https://github.com/openvinotoolkit/model_server.git 
+    branch: releases/2023/3 
+  dest:
+    url: https://github.com/opendatahub-io/openvino_model_server.git 
+    branch: 2023.3-release
+#Release following jobs
+- name: openvino_release
+  automerge: 'yes'
+  mode: 'release-following'
+  src:
+    url: https://github.com/openvinotoolkit/openvino.git
+    branch: releases/20*
+  dest:
+    url: https://github.com/opendatahub-io/openvino.git
+    branch: fast
+- name: openvino_contrib_release
+  automerge: 'yes'
+  mode: 'release-following'
+  src:
+    url: https://github.com/openvinotoolkit/openvino_contrib.git
+    branch: releases/20*
+  dest:
+    url: https://github.com/opendatahub-io/openvino_contrib.git
+    branch: fast

```

Comment:
@heyselbi, @Xaenalt and @rpancham can you confirm that this is the branch we went to create the new releases in ?

Reply:
Sure, that works for me

Reply:
fast branch works for me too!

Reply:
Cool. Then LGTM when this PR.